recipe-server/normandy/recipes/tests/api/v2/test_api.py

```diff
@@ -541,6 +542,32 @@ def test_list_filter_action(self, api_client):
             assert res.status_code == 200
             assert res.data['count'] == 0
 
+        def test_order_last_updated(self, api_client):
+            now = datetime.now()
+            yesterday = now - timedelta(days=1)
+            r1 = RecipeFactory(updated=yesterday)
+            r2 = RecipeFactory(updated=now)
+
+            res = api_client.get(f'/api/v2/recipe/?ordering=last_updated')
+            assert res.status_code == 200
+            assert [r['id'] for r in res.data['results']] == [r1.id, r2.id]
+
+            res = api_client.get(f'/api/v2/recipe/?ordering=-last_updated')
+            assert res.status_code == 200
+            assert [r['id'] for r in res.data['results']] == [r2.id, r1.id]
+
+        def test_order_name(self, api_client):
+            r1 = RecipeFactory(name="a")
+            r2 = RecipeFactory(name="b")
+
+            res = api_client.get(f'/api/v2/recipe/?ordering=name')
+            assert res.status_code == 200
+            assert [r['id'] for r in res.data['results']] == [r1.id, r2.id]
+
+            res = api_client.get(f'/api/v2/recipe/?ordering=-name')
+            assert res.status_code == 200
+            assert [r['id'] for r in res.data['results']] == [r2.id, r1.id]
+
```

Comment:
I think there should be a test that tests ordering by `xxxxx` or something. Will the API reject it or ignore it?

Reply:
The API should currently ignore it, though rejecting it would be nice.